ic-agent/build.rs

```diff
@@ -0,0 +1,61 @@
+use std::{env, fs, path::PathBuf};
+
+use candid::Principal;
+use ic_agent::Agent;
+
+const OUTPUT_DIR: &str = "OUT_DIR";
+// Generated Rust file storing API seed nodes.
+const OUTPUT_SEEDS_FILE: &str = "api_seed_nodes.rs";
+
+const MAINNET_ROOT_SUBNET_ID: &str =
+    "tdb26-jop6k-aogll-7ltgs-eruif-6kk7m-qpktf-gdiqx-mxtrf-vb5e6-eqe";
+
+// An initial seed list of API nodes used for fetching all existing API nodes.
+const API_SEED_NODES: [&str; 4] = [
+    "ic0.app",
```

Comment:
```suggestion
    "icp-api.io",
    "ic0.app",
    "icp0.io",
```

Reply:
We could yes, but should we? Will these domain work as seeds in the future? I was actually even reluctant to put `ic0.app` here, as it is not an API node.